front/app/containers/ProjectsShowPage/shared/survey/SurveyXact.tsx

```diff
@@ -70,14 +66,11 @@ export default memo<Props>(({ surveyXactUrl, className }) => {
           <Spinner />
         </Placeholder>
       )}
-      <Iframe
-        url={surveyXactUrl}
+      <iframe
+        src={surveyXactUrl}
         width={hackyWidthThingy}
-        height={
-          smallerThanLargeTablet ? surveyHeightMobile : surveyHeightDesktop
-        }
-        display={isIframeLoaded ? 'block' : 'none'}
-        overflow="hidden"
+        height={isSmallerThanTablet ? surveyHeightMobile : surveyHeightDesktop}
+        style={{ overflow: 'hidden' }}
```

Comment:
You missed the display style here, I think.

Reply:
It was intentional :-) I've tested in the browser and couldn't see a difference. But I've added it back to make sure I don't break anything and we'd be bug-fixing next week. It's only 1 line and doesn't hurt much.